content/developer-guide/identity-dev/dev-guide-least-privilege.md

```diff
@@ -2,13 +2,13 @@
 uid: LeastPrivilege
 ---
 
-# Least privileges
+# Least privilege
 
-When creating new users or client credential clients, configure them using the principle of Least Privilege. For more information, see the Cybersecurity & Infrastructure Security Agency article [Least Privilege](https://us-cert.cisa.gov/bsi/articles/knowledge/principles/least-privilege). Assign users and clients the lowest privileged role necessary to access resources.
+When creating new users or client-credentials clients, configure them using the principle of least privilege, which is that a process, user, or program should be able to access only the information and resources that are necessary for its legitimate purpose. Assign users and clients the lowest privileged role necessary to access resources.
+
+Ensure that clients which are not in use are either deleted or disabled by setting the **Enabled** property to false. This can be done through the portal (see <xref:ClientMaintenance>) or through the appropriate API (see <xref:identityClientCredentialClient>, <xref:identityHybridClient>, or <xref:identity-authorization-code-clients>).
```

Comment:
```suggestion
Ensure that clients that are not in use are either deleted or disabled by setting the **Enabled** property to false. This can be done through the portal (see <xref:ClientMaintenance>) or through the appropriate API (see <xref:identityClientCredentialClient>, <xref:identityHybridClient>, or <xref:identity-authorization-code-clients>).
```

Reply:
small grammar thing